### PR TITLE
Feature/nubis

### PR DESCRIPTION
**Pull request by gozer**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "MSQLPASS": "voicecommons",
   "MSQLDBNAME": "voiceweb",
   "MSQLHOST": "localhost",
-  "MYSQLPORT":  "3306",
+  "MYSQLPORT":  "3306"
```
Comment (mikehenrty) [inline config.json]: needs a comma here

File: package.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
     "walk": "^2.3.9"
   },
   "devDependencies": {
-    "@types/node": "^7.0.29",
+    "@types/node": "7.0.29",
```
Comment (mikehenrty) [inline package.json]: nice. we'll have to figure out what changed in 7.0.30+ later, but this is a good workaoround

File: config.json (hunk 1 above)
Comment (gozer) [inline config.json]: Wow, good catch! Fixed!

File: package.json (hunk 2 above)
Comment (gozer) [inline package.json]: Yeah, and Iv'e done the binary search, 7.0.30 is where things break

**Pull request merged**